src/main/src/test/java/org/geoserver/data/test/MockData.java

```diff
@@ -84,7 +84,7 @@
      */
```

Comment:
Whole bunch of nonfunctional whitespace changes in here, and no actual code changes. Try doing a `git checkout src/main/src/test/java/org/geoserver/data/test/MockData.java` and commit that.